### PR TITLE
Verify Alert Manager Is Shipped And Enabled Within ODS

### PR DESCRIPTION
**Pull request by pablofelix**
Signed-off-by: Pablo Felix Estevez Pico <pestevez@redhat.com>

### CODE REVIEW

**Review by lugi0 — APPROVED** [review]
lgtm

**Comment by pablofelix** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/827/

File: tests/Tests/100__deploy/100__installation/107__alert_manager.robot [hunk 1]
```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation     Post install test cases that verify OCP Alert Manager resources and objects
+Library           OpenShiftLibrary
+Resource          ../../../Resources/ODS.robot
+
+*** Test Cases ***
+Verify Alert Manager Is Shipped And Enabled Within ODS
```
Comment (bdattoma) [inline tests/Tests/100__deploy/100__installation/107__alert_manager.robot:7]: is there anything in the alertmanager configmap which might be worth to check here?

Comment (pablofelix) [reply tests/Tests/100__deploy/100__installation/107__alert_manager.robot:7]: In my opinion, alertmanager's configuration should have a test of its own, that could be added in another PR, does this make sense to you?

**Review by tarukumar — COMMENTED** [review]
Overall looks good. I would suggest to assign defined variable in Variable section  for namespace and name. 

Since those are used multiple times it will be easy to have control on future

File: tests/Tests/100__deploy/100__installation/107__alert_manager.robot (hunk 1 above)
Comment (jgarciao) [reply tests/Tests/100__deploy/100__installation/107__alert_manager.robot:7]: I agree with Pablo that this test suite can be enhanced in the future

**Review by jgarciao — APPROVED** [review]
lgtm

**Pull request merged**